lib/src/code_field/code_field.dart

```diff
@@ -56,7 +56,6 @@ class CodeField extends StatefulWidget {
   final bool lineNumbers;
 
   const CodeField({
-    Key? key,
```

Comment:
This package aims to be a drop-in replacement for https://pub.dev/packages/code_text_field
so we can get their user base when we go public.

For this reason, we must preserve as much of its public API as we can.

So we do need `key` here.

Moreover, `key` is very much desired for any public package because you never know how it will be used. Our rule to not require keys is limited to our job for Apache because it is not reused much. For this public package we will eventually raise the QA standard to this: https://github.com/alexeyinkin/dart-total-lints/blob/main/lib/package_2.18.yaml

And we also must keep the key argument like this and not replace it with `super.key` for less diff against https://github.com/BertrandBev/code_field/blob/master/lib/src/code_field/code_field.dart
Until we go public, this has higher priority than fixing linter issues.